src/components/ChatContainer/index.js

```diff
@@ -35,13 +34,32 @@ const enhance = compose(
     connect(mapStateToProps)
 );
 
-export const ChatContainer = ({ theme, connection }) => {
-    return (
-        <div className="ChatContainer">
-            <MessageList theme={theme} />
-            <InputArea {...theme} />
-        </div>
-    );
-};
+export class ChatContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            hasError: false
+        }
+    }
+
+    componentDidCatch(error, info) {
+        // TODO: Refactor this to dispatch a
+        // friendly message to the chat UI.
+        // Currently, we simply do not render the UI.
+        this.setState({ hasError: true });
+        console.error(error, info);
+    }
```

Comment:
Im fine with this for now we can abstract it later if/when we add multiple error boundries